src/main/java/training/supportbank/userinterface/UserInput.java

```diff
@@ -0,0 +1,90 @@
+package training.supportbank.userinterface;
+
+import training.supportbank.customexceptions.UserInputException;
+import training.supportbank.dataconverters.CsvConverter;
+import training.supportbank.dataconverters.JsonParser;
+import training.supportbank.models.Transaction;
+import training.supportbank.transactionhandlers.ExtractNames;
+import training.supportbank.transactionhandlers.Printer;
+import training.supportbank.transactionhandlers.ProcessTransactions;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+public class UserInput {

```

Comment:
Let's have a chat about the design of this class. We can probably make it a bit simpler.